doc/plutus/explanations/language-versions.rst

```diff
@@ -0,0 +1,23 @@
+.. _what_are_plutus_language_versions:
+
+What are Plutus language versions?
+==================================
+
+The Cardano ledger tags scripts with a *language*.
+This determines how the ledger will interpret the script, but also the *interface* to the script.
```

Comment:
What exactly is an "interface to a script"? It doesn't sound like a concept I'm familiar with

Reply:
I don't really know what to do when I want to refer to conceptual docs that don't exist yet. Maybe I should just write them.